### PR TITLE
Make MSLogger public again

### PR DESCRIPTION
**Pull request by MatkovIvan**

### CODE REVIEW

File: AppCenter/AppCenter/MSLogger.h [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #import <Foundation/Foundation.h>
 
 #import "MSConstants.h"
-#import "MSLogger.h"
```
Comment (MatkovIvan) [inline AppCenter/AppCenter/MSLogger.h]: OMG it was recursive. How did it work?

Comment (clpolet) [reply AppCenter/AppCenter/MSLogger.h]: Interesting...

File: AppCenter/AppCenter/MSLogger.h [hunk 2]
```diff
@@ -1,7 +1,6 @@
 #import <Foundation/Foundation.h>
```
Comment (clpolet) [inline AppCenter/AppCenter/MSLogger.h]: You can probably move the declaration of `msDefaultLogHandler` to the internal header.

Comment (clpolet) [reply AppCenter/AppCenter/MSLogger.h]: This header should also be part of the umbrella header.

**Pull request merged**